pygameMenu/widgets/widget/label.py

```diff
@@ -55,6 +55,8 @@ def _apply_font(self):
     # noinspection PyMissingOrEmptyDocstring
     def draw(self, surface):
         self._render()
+        if self._label != '':
```

Comment:
Why this condition `if self._label != '':` ? Even if there is no text it should be possible to se a background color/image?

Reply:
Yes, because the minimal surface width must be 1